### ISSUE TITLE
getOoxml on table leaves permanent selection highlight on page in Word

### ISSUE THREAD

**Issue by ciaran-vox** [OfficeDev/office-js#936]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

## Expected Behavior
When getting the Ooxml of a table, the word document does not change to include a permanent blue selection highlight

## Current Behavior
While I can get the ooxml ok, the word document adds a selection highlight to the table that does not disappear even after the table has been deleted or if I undo the action. Even when zooming in and out on the document the selection highlight remains the exact same size.

## Steps to Reproduce, or Live Example
Repro located here:
[https://github.com/ciaran-vox/WordMacBugRepro](https://github.com/ciaran-vox/WordMacBugRepro)

Or quickly add this function:
```
return Word.run(async context => {
        const selection = context.document.getSelection();  
        const tableOoxml = selection.parentTable.getRange().getOoxml();
        await context.sync(); 
        console.log(tableOoxml.value);     
});
```

## Environment
* Platform: Mac
* Host: Word v16.32
* Operating System: MacOS Catalina v10.15.2

* Platform: Windows
* Host: Word v1912 (Build 12325.20298)
* Operating System: Windows 10

## Useful logs
I've attached a screenshot of the problem.
![MacBug](https://user-images.githubusercontent.com/59871204/72341574-8029cd80-36c2-11ea-85ff-73160f476028.png)

This demonstrates what I mean when changing the document size but the selection highlight remains in place...
![MacBugZoom](https://user-images.githubusercontent.com/59871204/72342984-b7e64480-36c5-11ea-847a-543a0c54526b.png)

**Comment by ciaran-vox** [comment]
UPDATE: I previously said this does not occur on Windows, but it does. On Windows I was using Word version 1904 (Build 11601.20230) and this issue was not there. After an update (Windows not being my main environment) to Version 1912, the issue then presented itself. I've updated the issue title and initial post to reflect this.

UPDATE 2: I kept reverting back Office versions until the issue disappeared so I can pinpoint when the change occurred that introduced this issue. I found that the issue was introduced in version 1909 (Build 12026.20264).
